Source/Core/VideoBackends/D3D/D3DBase.cpp

```diff
@@ -17,61 +17,64 @@
 
 namespace DX11
 {
-HINSTANCE hD3DCompilerDll = nullptr;
-D3DREFLECT PD3DReflect = nullptr;
+using D3DREFLECT = HRESULT(WINAPI*)(LPCVOID, SIZE_T, REFIID, void**);
+
+static HINSTANCE s_d3d_compiler_dll = nullptr;
+static int s_d3dcompiler_dll_ref = 0;
+static D3DREFLECT s_d3d_reflect = nullptr;
 pD3DCompile PD3DCompile = nullptr;
-int d3dcompiler_dll_ref = 0;
 
 CREATEDXGIFACTORY PCreateDXGIFactory = nullptr;
-HINSTANCE hDXGIDll = nullptr;
-int dxgi_dll_ref = 0;
+static HINSTANCE s_dxgi_dll = nullptr;
+static int s_dxgi_dll_ref = 0;
 
-static D3D11CREATEDEVICE PD3D11CreateDevice = nullptr;
-HINSTANCE hD3DDll = nullptr;
-int d3d_dll_ref = 0;
+static D3D11CREATEDEVICE s_d3d11_create_device = nullptr;
+static HINSTANCE s_d3d_dll = nullptr;
+static int s_d3d_dll_ref = 0;
 
 namespace D3D
 {
 ID3D11Device* device = nullptr;
 ID3D11Device1* device1 = nullptr;
 ID3D11DeviceContext* context = nullptr;
-static IDXGISwapChain1* swapchain = nullptr;
-static ID3D11Debug* debug = nullptr;
-D3D_FEATURE_LEVEL featlevel;
-D3DTexture2D* backbuf = nullptr;
 HWND hWnd;
 
-std::vector<DXGI_SAMPLE_DESC> aa_modes;  // supported AA modes of the current adapter
+static IDXGISwapChain1* s_swapchain = nullptr;
+static ID3D11Debug* s_debug = nullptr;
+static D3D_FEATURE_LEVEL s_featlevel;
+static D3DTexture2D* s_backbuf = nullptr;
+
+static std::vector<DXGI_SAMPLE_DESC> s_aa_modes;  // supported AA modes of the current adapter
 
-bool bgra_textures_supported;
-bool allow_tearing_supported;
+static bool s_bgra_textures_supported;
+static bool s_allow_tearing_supported;
+static bool s_frame_in_progress = false;
+
+static unsigned int s_xres;
+static unsigned int s_yres;
 
 #define NUM_SUPPORTED_FEATURE_LEVELS 3
```

Comment:
This can likely be a `constexpr UINT`, it's only passed around as such for as far as I can see.